README.md

```diff
@@ -33,7 +33,7 @@ Version 1.3 (2016-09-27) of the OEM driver causes a kernel panic (a.k.a. *crash*
 
 * Install the driver by the following commands:
   * `brew tap adrianmihalko/ch340g-ch34g-ch34x-mac-os-x-driver https://github.com/adrianmihalko/ch340g-ch34g-ch34x-mac-os-x-driver`
-  * `brew cask install wch-ch34x-usb-serial-driver`
+  * `brew install cask wch-ch34x-usb-serial-driver`
```

Comment:
I understand that this is being changed due to `brew cask` being deprecated.

But does this do what you intend? Doesn't it merely install both `cask` and the `wch-ch34x-usb-serial-driver` package, rather than the cask version of `wch-ch34x-usb-serial-driver`?

So should this be `brew install --cask wch-ch34x-usb-serial-driver` instead?

If the _cask_ version isn't specifically required, then is `brew install wch-ch34x-usb-serial-driver` sufficient?

Reply:
I checked the console output and **cask** is not needed. I just installed a package called **cask.** 

` brew install wch-ch34x-usb-serial-driver` works fine.